otter/integration/lib/autoscale.py

```diff
@@ -317,25 +332,50 @@ def poll():
 
 
 @attributes([
-    Attribute('scale_by', instance_of=int),
+    Attribute('scale_by', default_value=None),
     Attribute('scaling_group', instance_of=ScalingGroup),
+    Attribute('set_to', default_value=None),
+    Attribute('scale_percent', default_value=None),
+    Attribute('name', instance_of=str, default_value='integration-test-policy')
 ])
 class ScalingPolicy(object):
     """ScalingPolicy class instances represent individual policies which your
-    integration tests can execute at their convenience.
+    integration tests can execute at their convenience. Only one of (scale_by,
+    set_to, scale_percent) should be provided. If more than one is provided,
+    this function will blindly include them in the policy creation request.
 
     :param int scale_by: The number of servers to scale up (positive) or down
         (negative) by.  Cannot be zero, lest an API-generated error occur.
     :param ScalingGroup scaling_group: The scaling group to which this policy
         applies.
+    :param int set_to: The number of servers to set as the desired capacity
+    :param float scale_percent: The percentage by which to scale the group up
+        (positive) or down (negative)
+    :param str name: A string to use as the name of the scaling policy. A
+        timestamp will be appended automatically for differentiation.
     """
 
     def __init__(self):
+
+        name_time = '{0}_{1}'.format(self.name,
+                                     datetime.datetime.utcnow().isoformat())
+        change_type = ""
+        change_factor = 0
+        if self.scale_by:
+            change_type = "change"
+            change_factor = self.scale_by
+        elif self.set_to:
+            change_type = "desiredCapacity"
+            change_factor = self.set_to
+        elif self.scale_percent:
+            change_type = "changePercent"
+            change_factor = self.scale_percent
+
         self.policy = [{
-            "name": "integration-test-policy",
+            "name": name_time,
             "cooldown": 0,
             "type": "webhook",
-            "change": self.scale_by
+            change_type: change_factor
```

Comment:
Non-blocking - might be easier to have `change_type` and `change_factor` as attributes instead, since having one override another and maybe taking priority seems more confusing.  I can file a ticket if you agree, or not if you don't agree.